ddtrace/ext/__init__.py

```diff
@@ -1,5 +1,6 @@
-class AppTypes(object):
+PYTHON_STACK = "python.stack"
```

Comment:
Looks like you aren't using it.

For the scope of this PR, let's stick to `ERROR_STACK`, and change it once we are good on the specification.